board.rb

```diff
@@ -0,0 +1,73 @@
+# class Board
+class Board
+
+  DIMENSION = 3
+
+  attr_accessor :size, :table
```

Comment:
If `:table` is open to writing, somebody could alter the array directly. What would the implications be for that?